package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "tensorboard",
+  "version": "1.14.0",
```

Comment:
Presumably we'll want some way to keep this number, at a minimum, in sync with our pip package version?  Maybe we can add a presubmit check that verifies that this number always matches version.py?

Reply:
We can do that or we can make this version preposterous like 0. This number is mostly useful only when we are publishing a npm package which we are not (note, `private: "true"`). Which would you prefer?